### ISSUE TITLE
Modify plot legends to account for ggplot2 update

### ISSUE THREAD

**Issue by padilla410** [NOAA-OCM/SWMPrExtension#34]
Most legends will need to have the legend spacing changed from `theme(legend.spacing.x = unit(-6, 'pt'))` to `theme(legend.spacing.x = unit(6, 'pt'))`.

`historical_range` will need `theme(legend.spacing.x = unit(2, 'pt'))`

**Comment by padilla410** [comment]
This issue will be closed in a future pull request

**Comment by swmpkim** [comment]
@padilla410 , @DaveEslinger , this hasn't made it into the CRAN version of SWMPrExtension yet, right? From the commit messages, it looks like it's in the master branch here on github; but I'm still having to add these commands interactively to make the legends work with the new `ggplot2`.

Here's the output of `sessionInfo()`, just for the record:
```
other attached packages:
[1] SWMPrExtension_0.3.16.9000 lubridate_1.7.4            SWMPr_2.3.0               
[4] zoo_1.8-4                  ggplot2_3.1.0   
```

**Comment by padilla410** [comment]
@swmpkim, that's correct. The most recent version is on my personal github (https://github.com/padilla410/SWMPrExtension), version 1.1.1. I need to submit a pull request to Dave.

**Comment by padilla410** [comment]
Resolved in v 1.1.1
